packages/httpd.rb

```diff
@@ -3,21 +3,22 @@
 class Httpd < Package
   description 'The Apache HTTP Server Project is an effort to develop and maintain an open-source HTTP server for modern operating systems including UNIX and Windows.'
   homepage 'http://httpd.apache.org/'
-  version '2.2.32'
-  source_url 'https://github.com/apache/httpd/archive/2.2.32.tar.gz'
-  source_sha256 '0d74b1e4d69cedc56323cedc819c083ca975b679b6a082abf60c21836756bb1d'
+  version '2.4.27'
+  source_url 'http://www-us.apache.org/dist/httpd/httpd-2.4.27.tar.gz'
+  source_sha256 '346dd3d016ae5d7101016e68805150bdce9040a8d246c289aa70e68a7cd86b66'
 
-  depends_on 'subversion'
+  depends_on 'apr'
+  depends_on 'aprutil'
+  depends_on 'libtool'
+  depends_on 'pcre'
 
   def self.build
-    system 'svn co http://svn.apache.org/repos/asf/apr/apr/branches/1.4.x srclib/apr'
-    system 'svn co http://svn.apache.org/repos/asf/apr/apr-util/branches/1.4.x srclib/apr-util'
-    system './buildconf'
-    system './configure --prefix=/usr/local'
+    system './configure'
     system 'make'
   end
 
   def self.install
-    system 'make -i install'
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "libtool --finish #{CREW_DEST_DIR}#{CREW_PREFIX}/apache2/modules"
```

Comment:
This last line shows:
```
libtool: finish: PATH="/usr/local/bin:/usr/bin:/bin:/opt/bin:/sbin" ldconfig -n /usr/local/tmp/crew/dest/usr/local/apache2/modules
----------------------------------------------------------------------
Libraries have been installed in:
   /usr/local/tmp/crew/dest/usr/local/apache2/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the '-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the 'LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the 'LD_RUN_PATH' environment variable
     during linking
   - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to '/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------
```
I'm just wondering we need this information?  I'm not sure how to use libtool, so this information may be correct and important.  I'm asking this question since I don't know about it.  Thanks.